docs/draft/17-url-scheme.md

```diff
@@ -0,0 +1,42 @@
+---
+permalink: /spec/17
+parent: Draft specs
+title: 17/URL SCHEME
+---
+
+# 17/Defined Url Scheme
+
+> Version: 0.1.0
+>
+> Status: Draft
+>
+> Authors: Aleksandar Djenic <aleksandardjenic@status.im>
+>
+
+
+## Table of Contents
+
+ 1. [Url Scheme](#url-scheme)
+ 2. [Defined urls](#defined-urls)
+
+## Url scheme
+
+- internal: "status-im://"
+- external: "https://join.status.im"
+
+## Defined urls
+
+| Name | Short Url | Long Url | Description |
+| ----- | ---- | ---- | ---- |
+| Browse | `/b/[url]` | `/browse/[url]`  | Opens `url` in the app's browser |
+| User profile | `/u/[user_pk &#124; ens_name]` | `/user/[user_pk &#124; ens_name]` | Displays user profile popup for user with `user_pk` or `ens_name` |
```

Comment:
What is this `&#124;`? Were you trying to add a vertical bar? Just write `or`.

Reply:
I couldn't use `|` in a cell and that's why I added it as html code. Changed it with `or`.

Reply:
Yeah, that won't work in most markdown parsers.